### ISSUE TITLE
Ahout audio pretrain model

### ISSUE THREAD

**Issue by gyt1145028706** [AryanShekarlaban/data2vec-pytorch#21]
Have you tested the WER (Word Error Rate) for the ASR task when fine-tuning the audio data2vec model trained with your code using CTC loss? Thanks !

**Comment by arxyzan** [comment]
Hi @gyt1145028706 , unfortunately not. I did not actually pretrain any of the models but just transported the weights. I highly recommend you checkout the main repo at FAIR, which also has the data2vec2 too.

**Comment by gyt1145028706** [comment]
Thanks !
